packages/client/lib/sections/Game.tsx

```diff
@@ -36,23 +32,24 @@ const Game = (props: Props) => {
     // If location already shot, alert pick another location
 
     // const hitStatus = checkHit(location, props.playerData);
-    const { index, shipHit } = checkHit(location, userData);
+    const { index, shipHit } = checkHit(location, playerData);
     console.log("index of the placedLocation: " + index);
     console.log("shipType hit: " + shipHit);
 
     // IF MISS
+
     if (index === -1) {
       //first number as row
       console.log("miss");
 
       // Make this piece of code reusable -> updates board and placedlocation
-      setData((prev) => {
-        const updatedBoard = prev.board.map((row, i) => {
-          if (i === x) {
-            return row.map((cell, j) => (j === y ? -1 : cell));
-          }
-          return row;
-        });
+      setPlayerData((prev) => {
+        // const updatedBoard = prev.board.map((row, i) => {
```

Comment:
I commented out parts of this function because it is broken with the change I've made. Fix for me ty :D